feign/src/test/java/com/baeldung/feign/clients/header/dynamicheader/BookClientLiveTest.java

```diff
@@ -0,0 +1,44 @@
+package com.baeldung.feign.clients.header.dynamicheader;
+
+import com.baeldung.feign.clients.builder.BookFeignClientBuilder;
+import com.baeldung.feign.models.Book;
+import lombok.extern.slf4j.Slf4j;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
+
+/**
+ * Consumes https://github.com/Baeldung/spring-hypermedia-api
+ */
+@Slf4j
```

Comment:
Let's not use Lombok to create loggers for tests.

Tests shouldn't log, and it makes the test code more oblique to add this.

Reply:
Removed SLF4J from these unit tests.